### PR TITLE
Upgrade to v3.25.0 of the AWS Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #1317
Fixes: #1318

### CODE REVIEW

File: sdk/dotnet/ElastiCache/Cluster.cs [hunk 1]
```diff
@@ -462,31 +463,22 @@ public InputList<string> SecurityGroupIds
...
-        /// Example: `arn:aws:s3:::my_bucket/snapshot1.rdb`
+        /// A single-element string list containing an Amazon Resource Name (ARN) of a Redis RDB snapshot file stored in Amazon S3. The object name cannot contain any commas. Changing `snapshot_arns` forces a new resource.
         /// </summary>
-        public InputList<string> SnapshotArns
-        {
-            get => _snapshotArns ?? (_snapshotArns = new InputList<string>());
-            set => _snapshotArns = value;
-        }
+        [Input("snapshotArns")]
+        public Input<string>? SnapshotArns { get; set; }
```
Comment (mikhailshilkov) [inline sdk/dotnet/ElastiCache/Cluster.cs:478]: This is breaking

**Comment by ericrudder** [comment]
i think the comment about policy would be great to add to the changelog

**Comment by stack72** [comment]
> i think the comment about policy would be great to add to the changelog
> […](#)


It's already in there @ericrudder

**Comment by ericrudder** [comment]
i didn't see this in the changelog diffs ... i saw the change, not the
"explanation"

if i fat fingered the search, please LMK

**Comment by stack72** [comment]
it's already in master :)

**Comment by ericrudder** [comment]
you are a gentleman and a scholar!

**Review by mikhailshilkov — APPROVED** [review]

**Pull request merged**